header.php

```diff
@@ -19,30 +19,35 @@
 <?php wp_head(); ?>
 </head>
 
-<body <?php body_class(); ?>>
+<body <?php body_class(); _s_schema_microdata(); ?>>
 <div id="page" class="site">
 	<a class="skip-link screen-reader-text" href="#content"><?php esc_html_e( 'Skip to content', '_s' ); ?></a>
 
-	<header id="masthead" class="site-header" role="banner">
-		<div class="site-branding">
+	<header id="masthead" class="site-header" role="banner" itemscope="itemscope" itemtype="http://schema.org/WPHeader">
+		<div class="site-branding" itemscope itemtype="http://schema.org/Organization">
```

Comment:
I'm trying to add it to same .site-branding but I'm still experimenting, as there are no other good places to add it when marking Article:

https://developers.google.com/search/docs/data-types/articles